.eslint-ci.config.js

```diff
@@ -21,12 +24,14 @@ module.exports = {
       'error',
       {
         devDependencies: [
+          'jest*.[jt]s',
```

Comment:
Why do we need this? Is this a preparation for an upcoming change that uses module in the config files?